### DISCUSSION TITLE
Monthly ESMValTool community meeting December 2022

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#66] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next monthly ESMValTool community meeting will be in December. The result of the poll is that the meeting will take place on

Friday, December 16 at 11 AM CET

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we choose some topics to talk about in more detail.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in https://github.com/ESMValGroup/Community/discussions/60#discussioncomment-4148411

**Comment by katjaweigel** [comment]
The meeting code does not work?

**Reply by bouweandela** [reply]
Sorry, you should be able to join now without a passcode

**Reply by catherinehardacre** [reply]
I also can't access the meeting...

**Comment by tepmo42** [comment]
Thanks! That worked  - I couldn't see a code

**Comment by bouweandela** [comment]
Personal updates
----------------

- Twan van Noije from KNMI joined, working on EC-earth model development
- @glpotter Working on CMORizing reanalysis data for CREATE-IP: questions about MERRA2
- @rswamina Would like to discuss handling observational datasets #70, question about daily precipitation data 
- @remi-kazeroni Will be the release manager for the upcoming v2.8, question about iris versions
- Amy Doherty from Met Office joined, interested in discussing observational data storage
- @PaulTGriffiths from NCAS joined, interested in developing diagnostics
- @katjaweigel Working on the flato recipe, questions about datasets
- Catherine Hardacre from the Met Office joined, developing a recipe to compare model output with observations of aerosol optical depth from AERONET
- Tamzin Palmer Implementing Met Office diagnostics in ESMValTool, has some issues with miles blocking recipe because of daily ERA5 data
- @valeriupredoi joined 
- @axel-lauer Working on adding new cloud diagnostics and reviewing IPCC diagnostics

**Reply by catherinehardacre** [reply]
I forgot to say: I recently visited the National Institute for Water and Atmospheric research (NIWA, a Met Office partner) and did some ESMValTool outreach. I think I managed to recruit at least one new user!

**Comment by bouweandela** [comment]
@glpotter reported that MERRA2 vertical levels have the wrong units (should be Pascals)

**Reply by bouweandela** [reply]
@glpotter Could you please [open a GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new/choose) to report this?

**Comment by bouweandela** [comment]
List of reanalysis datasets supported by ESMValTool posted by @remi-kazeroni: https://github.com/ESMValGroup/ESMValTool/issues/2851

**Comment by bouweandela** [comment]
@rswamina Would like to work with daily precipitation data TRMM. Some is available in obs4MIPs on ESGF
@glpotter suggests using new and better data from IMERG, which he mentions is available somewhere. Could you add a bit more information here @glpotter?

**Reply by rswamina** [reply]
The IMERG raw data is available [here](https://gpm.nasa.gov/data/directory).

**Comment by bouweandela** [comment]
@remi-kazeroni A bugfix release of ESMValCore v2.7.1 was made to enable using the latest version of iris with the ESMValTool when installing that from source. Note that the released ESMValTool package will still use an older version of iris because that is what it was tested with.

**Comment by bouweandela** [comment]
Recipe to produce CMORized daily ERA5 data: https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/cmorizers/recipe_daily_era5.yml
